src/settings.rs

```diff
@@ -528,6 +557,42 @@ impl Index {
         .await
     }
 
+    /// Update [pagination](https://docs.meilisearch.com/learn/configuration/settings.html#pagination) of the [Index].
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use meilisearch_sdk::{client::*, indexes::*, settings::Settings};
+    /// #
+    /// # let MEILISEARCH_HOST = option_env!("MEILISEARCH_HOST").unwrap_or("http://localhost:7700");
+    /// # let MEILISEARCH_API_KEY = option_env!("MEILISEARCH_API_KEY").unwrap_or("masterKey");
+    /// #
+    /// # futures::executor::block_on(async move {
+    /// let client = Client::new(MEILISEARCH_HOST, MEILISEARCH_API_KEY);
+    /// # client.create_index("set_pagination", None).await.unwrap().wait_for_completion(&client, None, None).await.unwrap();
+    /// let mut index = client.index("set_pagination");
+    /// let mut pagination = std::collections::HashMap::new();
+    /// pagination.insert(String::from("maxTotalHits"), 100);
+    /// let task = index.set_pagination(&pagination).await.unwrap();
+    /// # index.delete().await.unwrap().wait_for_completion(&client, None, None).await.unwrap();
+    /// # });
+    /// ```
+    pub async fn set_pagination(
+        &self,
+        pagination: &HashMap<String, i32>,
```

Comment:
I think it would be best to create a `PaginationSetting` struct to handle this, in order for the users to know what the options are.

Reply:
@bidoubiwa Please review.

Reply:
I will review as soon as I have time :)

Reply:
haha..sure